jsk_robot_common/jsk_robot_startup/lifelog/common_logger.launch

```diff
@@ -1,4 +1,8 @@
 <launch>
+  <!-- mongo setting -->
+  <arg name="database" default="jsk_robot_lifelog" />
+  <arg name="collection" default="null" />
```

Comment:
does this `null` work properly?
I'm just curious.
Have you checked with this launch with default value?

Reply:
No, not yet. I tested with arguments such as `fetch15` and `go_to_kitchen`.
Actually, rostest fails in this section.

Reply:
So, I think another default value should be set like `(robot_name)` or something.

Reply:
I changed default collection to `robot` [ed96b54](https://github.com/jsk-ros-pkg/jsk_robot/pull/1719/commits/ed96b54764c5ea8236efa5ef0968486147ae6c58)

Reply:
I discussed with @knorth55, we support `""` argument by using eval" 
2f347eb